bin/dev

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+if ! command -v foreman &> /dev/null
+then
+  echo "Installing foreman..."
+  gem install foreman
```

Comment:
do we not want this as part of the Gemfile?

Reply:
Rails 7 default